src/dawn/Optimizer/PassStageSplitter.cpp

```diff
@@ -28,7 +28,7 @@
 
 namespace dawn {
 
-PassStageSplitter::PassStageSplitter() : Pass("PassStageSplitter") {}
+PassStageSplitter::PassStageSplitter() : Pass("PassStageSplitter") { isDebug_ = true; }
```

Comment:
minor thing, but would be better to have a ctr initialization
```, isDebug_(true){}```
Also in PassMultiStageSplitter.cpp
or pass it to the Pass ctr `Pass("name", true)` and then it could be set `const bool`

Reply:
ctr does not work as it is a protected member of the base-class.

We can go the second route if you think it is prettier

Reply:
done